addons/web/static/src/views/kanban/kanban_header.js

```diff
@@ -107,8 +107,8 @@ export class KanbanHeader extends Component {
         } else if (!name) {
             if (
                 isRelational(groupByField) ||
-                groupByField.type === "date" ||
-                groupByField.type === "datetime" ||
+                groupByField.value === undefined ||
+                groupByField.displayName === false ||
```

Comment:
I think groupByField is the definition of the field (def fields_get). This change doesn't seem to mean anything
as value and display name won't be present if i'm right